docs/source/dummies_participant.rst

```diff
@@ -0,0 +1,43 @@
+submitted - the participant has finished the experiment and submitted their work. The work is now pending approval (MTurk requires the experimenter to approve or reject work before the participant is paid). Dallinger auto-approves all submissions so status should not remain as submitted for very long. If it stays like this for more than a minute chances are there is a bug in your experiment and you should expect emails from Turk workers wondering when they will be paid.
+approved - the work has been approved and the participant has been paid. This is good and suggests things are going to plan.
+rejected - by default Dallinger never rejects work, so if you are seeing this something unexpected is going on.
+returned - this means the participant quit half way through. A few participants will always change their mind and quit, but if you're getting a lot of these there's probably a bug and participants are quitting out of frustration. Mechanical turk automatically recruits a replacement participant when a participant returns the experiment.
+abandoned - mechanical turk gives participants a fixed amount of time in which to complete the study. If they time out they are kicked out and their status is set to abandoned. Lots of turk workers open the experiment but never do anything, so you'll see a steady stream of these.  Mechanical turk automatically recruits a replacement participant when a participant abandons an experiment.
+did_not_attend - Dallinger allows experiment to check that participants are paying attention (see the demos for examples of this). If a participant fails this check their status will be set to this value.
+bad_data - Dallinger also allows experiments to check that a participants data is correct (the right number of nodes etc). If a participant fails this check their status will be set to bad_data. A few such cases are pretty much unavoidable - participants do strange and reckless things. But if you're getting a lot of these then there's probably a deeper issue. Unless instructed otherwise, Dallinger will automatically recruits a replacement participant.
+missing_notification - Occasionally the communication system between Dallinger and Mechanical Turk fails. This can cause serious problems, but Dallinger now checks for this. The most comon case is that a participant returns or abandons but Mechanical Turk never informs us. When this happens the relevant participant will be set to missing_notification.
+replaced - Another oddity with Mechanical Turk is that sometimes it recycles specific assignments if a participant returns it very shortly after accepting it. Again, Dallinger now tries to catch these instances and uses the status replaced to mark affected participants.
+
```

Comment:
We now have `overrecruited`, also, to mark a participant who accepted a HIT which is already at capacity.

Reply:
@thomasmorgan Maybe I'm wrong about that, and maybe this is why the build is erroring immediately on Travis. Any reason to no rebase onto master now?

Reply:
@jessesnyder I think its out of date, the last commit prior to opening the branch was f307f7ad9925841597e06de09dec27201dd3b4e4. Rebasing now should be fine - the branch only adds new files and does not edit any existing ones. Though the main reason not to do so is my lack of proficiency in these sorts of things. Can you handle the rebasing? Alternatively, if you give me some hints I can probably figure it out (e.g. do I need to undo the PR before rebasing?)

Reply:
@thomasmorgan Sure, I'll try the rebase. I'll let you know when I've pushed the result.

Reply:
Well, that turned out to be easy! Let's see how the Travis build goes.